Formula/pict.rb

```diff
@@ -0,0 +1,24 @@
+class Pict < Formula
+  desc "Pairwise Independent Combinatorial Tool"
+  homepage "https://github.com/Microsoft/pict/"
+  url "https://github.com/Microsoft/pict/archive/v3.7.1.tar.gz"
+  sha256 "4fc7939c708f9c8d6346430b3b90f122f2cc5e341f172f94eb711b1c48f2518a"
+
+  def install
+    system "make"
+    bin.mkpath
+    bin.install "pict"
+  end
+
+  test do
+    system "test", "#{bin}/pict"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
After that line I effectively test that the cli command runs coorectly.

Reply:
Just check that it outputs its own name is not sufficient. We need to exercise some functionality please.

Reply:
ok...I will improve this

Reply:
Improved